cmd/livepeer/starter/starter.go

```diff
@@ -1105,30 +1115,28 @@ func StartLivepeer(ctx context.Context, cfg LivepeerConfig) {
 			for _, config := range configs {
 				modelConstraint := &core.ModelConstraint{Warm: config.Warm}
 
-				// Set price per unit base info.
-				pixelsPerUnit, ok := new(big.Rat).SetString(*cfg.PixelsPerUnit)
-				if !ok || !pixelsPerUnit.IsInt() {
-					panic(fmt.Errorf("-pixelsPerUnit must be a valid integer, provided %v", *cfg.PixelsPerUnit))
-				}
-				if pixelsPerUnit.Sign() <= 0 {
-					// Can't divide by 0
-					panic(fmt.Errorf("-pixelsPerUnit must be > 0, provided %v", *cfg.PixelsPerUnit))
-				}
-				pricePerUnit, currency, err := parsePricePerUnit(config.PricePerUnit.String())
-				if err != nil {
-					panic(fmt.Errorf("'pricePerUnit' value specified for model '%v' in pipeline '%v' must be a valid integer with an optional currency, provided %v", config.ModelID, config.Pipeline, config.PricePerUnit))
-				} else if pricePerUnit.Sign() < 0 {
-					panic(fmt.Errorf("'pricePerUnit' value specified for model '%v' in pipeline '%v' must be >= 0, provided %v", config.ModelID, config.Pipeline, config.PricePerUnit))
-				}
-				pricePerPixel := new(big.Rat).Quo(pricePerUnit, pixelsPerUnit)
 				var autoPrice *core.AutoConvertedPrice
-				if *cfg.Network == "offchain" {
-					autoPrice = core.NewFixedPrice(pricePerPixel)
-				} else {
-					autoPrice, err = core.NewAutoConvertedPrice(currency, pricePerPixel, nil)
-				}
-				if err != nil {
-					panic(fmt.Errorf("error converting price: %v", err))
+				if *cfg.Network != "offchain" {
```

Comment:
@victorges I now added simple if conditions since I didn't want to cause even more conflicts with the already excisting pull requests that are on the `ai-video` branch. I however would like to improve this logic in the future so I created https://linear.app/livepeer-ai/issue/AI-546/improve-ai-configuration-parsing-logic-in-startergo.

Reply:
@ad-astra-video already did some improvements in the AI remote worker implementation https://github.com/livepeer/go-livepeer/pull/3088. Maybe we can merge this ugly variant and handle it later?

Reply:
So the idea is we ignore any pricing config when it's offchain, is that it? I guess it makes sense.